spec/suites/ips/condition_spec.rb

```diff
@@ -1,6 +1,6 @@
 RSpec.describe IPS::Condition do
   let(:suite) { Inferno::Repositories::TestSuites.new.find('ips') }
-  let(:group) { suite.groups[1].groups[21] }
+  let(:group) { suite.groups.first.groups[1].groups[3] }
```

Comment:
nit to consistently use `first`. take it or leave it.

```suggestion
  let(:group) { suite.groups.first.groups.first.groups[3] }
```

Reply:
Just kidding I can't count.